index.html

```diff
@@ -18,13 +18,14 @@ <h1 class="page-title-text">
         </div>
         <div class="branding-definition">
             <h1 class="branding-definition-text">
```

Comment:
> Normalize vs reset css. I read about both of them, I think I still don't understand when to use one or the other. 

This article is one of the best explaining the difference: https://thisthat.dev/normalize-vs-reset-css/

Reset removes all the styles by default. Normalize, makes the built-in browser styles consistent across browsers.

Eg: h1.
- reset will display h1 with a default text size and without margins:
- normalize will display h1 with 2em text size, bold and some margins (similar to most browsers)

Try both, you can even look at the source code of each (normalize and reset) and see the differences yourself.


> Plus, do we always have to choose them for every project? What's your best choice?

You don't need to use them, but you should for styling consistency across browsers.

Usually I use a modern version of reset: [minireset](https://jgthms.com/minireset.css/) because it has the very minimum of the resets I use in all my projects.

Reply:
Is the aim of reset css or normalize css to set a kind of default css in case the css you coded doesn't display for some reason?

Reply:
Not really
- reset: Remove some browser built-in styles for you (ex remove h1 margins), so that you don't need to do it yourself.
- normalize: Making sure the default h1 renders equally the same across all browsers, if youu don't style it.